session/pessimistic_test.go

```diff
@@ -44,8 +44,8 @@ type testPessimisticSuite struct {
 
 func (s *testPessimisticSuite) SetUpSuite(c *C) {
 	testleak.BeforeTest()
-	// Set it to 300ms for testing lock resolve.
-	tikv.PessimisticLockTTL = 300
+	// Set it to 15s for testing lock resolve.
+	tikv.PessimisticLockTTL = 15000
```

Comment:
Can we decrease ticker interval for test instead of increase TTL?
So the test can run faster.

Reply:
If `tikv.PessimisticLockTTL` is set to 300ms, and ttlManager ticker is 3s

The lock of the pessimistic txn would be resolved before ttlManager take effect:
https://github.com/pingcap/tidb/pull/12177/files/9f5e3aa4c6402e4298686f660462edb1d6b58eed#diff-d76c0e3a34d441abfd93e7d3995d46e5R374

Reply:
Or should we change here and change the test behavior?